### PR TITLE
CSS Four Column Layout First Draft

### PR DESCRIPTION
**Pull request by neldaza**
Screen Shot 2021-10-14 at 12 48 29 AM

### CODE REVIEW

File: css-four-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    </div>
+    <div class="footer-column three">
+      <h3>Service Filters</h3>
+      <p>Accepts only your ideal trips.</p>
+    </div>
+    <div class="footer-column four">
+      <h3>Stats + Mileage</h3>
+      <p>Tracks your trips across all apps.</p>
+    </div>
+  </div>
```
Comment (codingcodymiller) [inline css-four-column-layout/index.html:55]: I can see why you broke these pieces of the layout into rows based on how they look in terms of "layout", but this makes the content of each column seem separate from one another. You should have one `row` with four `columns`, each of which holds the image, icon, and text for that column

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Good start! One note to get you started:

File: css-four-column-layout/index.html (hunk 1 above)
Comment (neldaza) [reply css-four-column-layout/index.html]: revised!

File: css-four-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+.three {
+  position: relative;
+  left: 0.1rem;
+}
+
+.four {
+  position: relative;
+  left: 0.1rem;
+}
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css:74]: You should just make this into two reusable classes, the declaration block is duplicated once for each of these

File: css-four-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  width: 80%;
+  margin: auto;
+  background-color: #f7f8fa;
+  padding-left: 8rem;
+  padding-right: 8rem;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+}
+
+.header-row {
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css:11]: You should make a `row` class with just `display: flex` and make this an additional class you could add along with that class. Try to stick to the plan for layout classes, have `container`, `row`, and `column`, all with just the styling they need for their purpose, keep them reusable.

**Comment by neldaza** [comment]
new output with said revisions!


Screen Shot 2021-10-16 at 10 14 49 PM
:

**Pull request merged**